ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/input-variables.md

```diff
@@ -1043,10 +1043,10 @@ roles/eos_cli_config_gen/docs/tables/platform.md
 roles/eos_cli_config_gen/docs/tables/poe.md
 --8<--
 
-### PTP
+### PTP settings
```

Comment:
Please revert this change, and update the eos_designs input variables documentation.
Additionally, I beleive we would want to keep both `ptp` and `ptp_settings`, so that users have the deprecation documentation.

Reply:
Reverted this and updated eos_designs input-variables.md.

Reply:
I am suggesting the opposite. A single documentation section, with a single table, but that table contains both the new and old key. That is how we do elsewhere.

Reply:
I added documentation of both new and old key in a single markdown file. Please review again.